readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://xXPaRaZorXx.github.io/layout_stars/)
+- [TEST REPORT LINK](https://xXPaRaZorXx.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to PR description
<img width="636" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/2216004d-4d39-4b4b-b295-e9e022b5d117">